engines/bastion_katello/app/assets/javascripts/bastion_katello/content-hosts/bulk/views/content-hosts-bulk-environment-modal.html

```diff
@@ -3,7 +3,12 @@ <h4 data-block="modal-header" translate>Content Host Bulk Content</h4>
 
   <div data-block="modal-body">
     <h4 translate>Assign Lifecycle Environment and Content View</h4>
-
+    <p>Please run the following commands on the clients, so the new Content View can be picked up.
+    <br/>
+    # yum clean all
+    <br/>
+    # subscription-manager refresh
+    </p>
```

Comment:
I recommend the [`<pre>`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/pre) element here, since it's displaying code.  It'll give you built-in styling (and also remove the need for the `<br>`'s)